### PR TITLE
Fix rllib related documentation and examples.

### PR DESCRIPTION
**Pull request by Gamenot**
See changelog.

### CODE REVIEW

**Comment by Gamenot** [comment]
Failing docs test is expected because `{ppo_example|ppo_pbt_example}.py` do not exist yet on `master`.

File: docs/ecosystem/rllib.rst [hunk 1]
```diff
@@ -6,17 +6,17 @@ RLlib
 
 **RLlib** is an open-source library for reinforcement learning that offers both high scalability and a unified API for a variety of applications. ``RLlib`` natively supports ``TensorFlow``, ``TensorFlow Eager``, and ``PyTorch``. Most of its internals are agnostic to such deep learning frameworks.
 
-SMARTS contains two examples using `Policy Gradients (PG) <https://docs.ray.io/en/latest/rllib-algorithms.html#policy-gradients-pg>`_.
+SMARTS contains two examples using `Proximal Policy Optimization (PPO) <https://docs.ray.io/en/latest/rllib/rllib-algorithms.html#ppo>`_.
 
 #. Policy gradient
```
Comment (Adaickalavan) [inline docs/ecosystem/rllib.rst:11]: PPO

File: docs/ecosystem/rllib.rst [hunk 2]
```diff
@@ -6,17 +6,17 @@ RLlib
...
 
-    + script: :examples:`e12_rllib/pg_example.py`
+    + script: :examples:`e12_rllib/ppo_example.py`
     + Shows the basics of using RLlib with SMARTS through :class:`~smarts.env.rllib_hiway_env.RLlibHiWayEnv`.
 
 #. Policy gradient with population based training
```
Comment (Adaickalavan) [inline docs/ecosystem/rllib.rst:16]: PPO with population based training

File: docs/ecosystem/rllib.rst (hunk 1 above)
Comment (Gamenot) [reply docs/ecosystem/rllib.rst:11]: I pushed a fix to these on `master`.

**Pull request merged**